ix-dev/test/glances/app.yaml

```
@@ -0,0 +1,42 @@
+app_version: 6.7.1
+capabilities:
+- description: Glances is able to bypass permission checks for it's sub-processes.
+  name: FOWNER
+- description: Glances is able to bypass permission checks.
+  name: DAC_OVERRIDE
+- description: Glances is able to bypass permission checks.
+  name: SETGID
+- description: Glances is able to bypass permission checks.
+  name: SETUID
+- description: Glances is able to trace system calls.
+  name: SYS_PTRACE
+categories:
+- monitoring
+description: Glances is a cross-platform system monitoring tool.
+home: https://nicolargo.github.io/glances/
+host_mounts: []
+icon: https://media.sys.truenas.net/apps/glances/icon.svg
+keywords:
+- metric
+- monitoring
+lib_version: 2.1.8
+lib_version_hash: a3251fe8d434248fc48ce7e1a29de015dab56de3687a93b925e79c986db2be44
+maintainers:
+- email: dev@ixsystems.com
+  name: truenas
+  url: https://www.truenas.com/
+name: glances
+run_as_context:
+- description: Glances runs as root user.
+  gid: 0
+  group_name: root
+  uid: 0
+  user_name: root
+screenshots:
+- https://media.sys.truenas.net/apps/glances/screenshots/screenshot1.png
+- https://media.sys.truenas.net/apps/glances/screenshots/screenshot2.png
+sources:
+- https://hub.docker.com/r/nicolargo/glances
+title: Glances
+train: test
+version: 0.0.1
```

Please move it to community train. Test is mostly used for enterprise apps to be tested.

```suggestion
train: community
version: 1.0.0
```